mapper/thin_edge_json/src/measurement.rs

```diff
@@ -164,19 +62,19 @@ pub trait FlatMeasurementVisitor {
 ///     }
 /// }
 /// ```
-pub trait GroupedMeasurementVisitor {
-    /// Error type specific to this way of collecting measurements
+pub trait MeasurementVisitor {
+    /// Error type specific to this visitor.
     type Error: std::error::Error + std::fmt::Debug;
 
-    /// Set the timestamp shared by all the measurements of this serie
-    fn timestamp(&mut self, value: DateTime<FixedOffset>) -> Result<(), Self::Error>;
+    /// Set the timestamp shared by all the measurements of this series.
+    fn visit_timestamp(&mut self, value: DateTime<FixedOffset>) -> Result<(), Self::Error>;
```

Comment:
This is a matter of taste, but I prefer the name `timestamp` without the `visit` prefix!

What matters is that the team is comfortable with the naming,

Reply:
IMHO, it makes it clearer what are the callback functions and what are functions that do not belong to the Visitor trait. For instance, the `done` or `finish` methods are not defined in the trait, and these are the methods that should be called directly by the consumer, whereas the callback functions should not be called directly.

Reply:
NOTE that `serde` also prefixes all visitor methods with `visit_` and I can point you at various other implementations that do exactly the same. I understand it's mostly taste, but there is some further reasoning behind these changes, mainly that methods should name actions.